examples/test-reader.py

```diff
@@ -51,8 +51,13 @@ def write(fstr, *vals):
 #>> row={}
 #""", np.array_repr(j))
         gid, step, offset = j[0], j[1], j[2]
-        double_v = j[3] if i.double_n_val else None
-        long_v = j[4] if i.long_n_val and i.double_n_val else j[3] if i.long_n_val else None
+
+        double_v, long_v = {(False, False): lambda: (None, None),
```

Comment:
How about this?

Reply:
To test this, which sha on NEST should I use to produce the *.sion?

Reply:
Your solution works, I have just tested it with an existing sionlib output file in which all cases are represented.

Reply:
Btw, the SHA is 7aa979e8d0e2d30.
But then you have to write a simulation script with the spin_detector to cover all cases, and for the spin_detector you need special neurons to connect to it.